src/components/Canvas.tsx

```
@@ -909,12 +819,66 @@ const CanvasFlow = () => {
+      viewport: reactFlowInstance?.getViewport(),
+      metadata: {
+        projectId,
+        exportedAt: new Date().toISOString(),
+        nodeCount: nodes.length,
+        edgeCount: edges.length
+      }
+    };
+
+    switch (format) {
+      case 'json':
+        const dataStr = JSON.stringify(exportData, null, 2);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)